src/components/FilterComponents/poketype/PokeType.js

```diff
@@ -1,3 +1,21 @@
 /**
- * This dropdown filtres different types of pokemons.
- */
\ No newline at end of file
+ * 
+ */
+ import './PokeType.css';
+ 
+ function PokeType () {
+     return (
+     <div className="dropdown">
+         <br />
+         <p>Pokemon types</p>
+         <select className="something" value='All'>
```

Comment:
please create a state which is used by this dropdown. and  if you choose an option, the chosen option should be the selected one (so the value of the select)